dmt/main/models.py

```diff
@@ -958,6 +958,15 @@ def users_active_between(self, start, end):
 
         return active_users
 
+    def user_estimated_time_report(self):
+        """Returns a project report that lists user stats."""
+        return User.objects.filter(
```

Comment:
Should we remove items that are in the `Someday/Maybe` milestone? Should only OPEN and INPROGRESS items be included?

Reply:
According to Shashi, no - he wants everything included.

Reply:
Hmmm.. maybe it would make sense to ignore Someday/Maybe milestones. these special cases are annoying, i will ask him that. But from my understanding, closed items should be reflected here. Maybe @zmustapha has more info.

Reply:
In the Capacity report, the request was to ignore Someday/Maybe and only include OPEN/INPROGRESS. Let's definitely sync up on this so the numbers are somewhat aligned.

Reply:
We may have to clarify with Shashi. He didn't give me further info. I will follow up.

Reply:
Thanks, I'll make this change. I think the way I was calculating this was incorrect anyways.

Reply:
Take a look here: https://github.com/ccnmtl/dmt/blob/master/dmt/main/models.py#L126. That's where I started for capacity.